bitnami/opencart/templates/deployment.yaml

```diff
@@ -1,162 +1,241 @@
-        - name: SMTP_HOST
-          value: {{ default "" .Values.smtpHost | quote }}
-        - name: SMTP_PORT
-          value: {{ default "" .Values.smtpPort | quote }}
-        - name: SMTP_USER
-          value: {{ default "" .Values.smtpUser | quote }}
-        - name: SMTP_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ template "opencart.fullname" . }}
-              key: smtp-password
-        - name: SMTP_PROTOCOL
-          value: {{ default "" .Values.smtpProtocol | quote }}
-        ports:
-        - name: http
-          containerPort: 80
-        - name: https
-          containerPort: 443
-        livenessProbe:
-          httpGet:
-            path: /admin/
-            port: http
-            httpHeaders:
-            - name: Host
-              value: {{ include "opencart.host" . | quote }}
-          initialDelaySeconds: 120
-          timeoutSeconds: 5
-          failureThreshold: 6
-        readinessProbe:
-          httpGet:
-            path: /admin/
-            port: http
-            httpHeaders:
-            - name: Host
-              value: {{ include "opencart.host" . | quote }}
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-          periodSeconds: 5
-        resources:
-{{ toYaml .Values.resources | indent 10 }}
-        volumeMounts:
-        - name: opencart-data
-          mountPath: /bitnami/opencart
-{{- if .Values.metrics.enabled }}
-      - name: metrics
-        image: {{ template "opencart.metrics.image" . }}
-        imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
-        command: [ '/bin/apache_exporter', '--scrape_uri', 'http://status.localhost:80/server-status/?auto']
-        ports:
+        - name: {{ include "common.names.fullname" . }}
+          image: {{ template "opencart.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.image.debug | quote }}
+            - name: ALLOW_EMPTY_PASSWORD
+              value: {{ .Values.allowEmptyPassword | quote }}
+            - name: APACHE_HTTP_PORT_NUMBER
+              value: {{ .Values.containerPorts.http | quote }}
+            - name: APACHE_HTTPS_PORT_NUMBER
+              value: {{ .Values.containerPorts.https | quote }}
+            {{- if .Values.mariadb.enabled }}
```

Comment:
I think we could follow this approach 
- deployment https://github.com/bitnami/charts/blob/0ffded5c0785040097d76d38a20f15b2f4ae1ea4/bitnami/drupal/templates/deployment.yaml#L81
- helpers https://github.com/bitnami/charts/blob/0ffded5c0785040097d76d38a20f15b2f4ae1ea4/bitnami/drupal/templates/_helpers.tpl#L53

Reply:
Changed to follow the same approach as Drupal